### PR TITLE
set up confirmation page for new custos/oidc users

### PR DESCRIPTION
**Pull request by juleengraham**
Address part of #10075, asks user to confirm creation of new account to avoid duplicate accounts

### CODE REVIEW

File: client/src/components/login/Confirmation.vue [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+                </b-alert>
+                <b-alert :show="messageShow" :variant="messageVariant" v-html="messageText" />
+                <b-form id="confirmation" @submit.prevent="submit()">
+                    <b-card no-body header="Confirm new account creation">
+                        <b-card-body>
+                            <p>Looks like you are about to create a new account!</p>
+                            <p>
+                                Do you already have a Galaxy account?
+                                If you already have an account, hit <em>'No, go back'</em> to log in using your username 
+                                and password to connect this account via <strong>User Preferences</strong>.
```
Comment (afgane) [inline client/src/components/login/Confirmation.vue]:
```suggestion
                                and password to connect this account via <strong>User Preferences</strong>. <a href="https://galaxyproject.org/authnz/use/oidc/idps/custos/#link-an-existing-galaxy-account" target="_blank">See here</a> how.
```

File: client/src/components/login/Confirmation.vue [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+                            </p>
+                            <p>If you wish to continue with the new account, select <em>'Create new account'</em>.</p>
+                            <p>
+                                Reminder: Registration and usage of multiple accounts is tracked and such accounts are subject
+                                to termination and data deletion. Connect existing account now to avoid possible loss of data.
```
Comment (afgane) [inline client/src/components/login/Confirmation.vue]:
```suggestion
                                to termination and data deletion on public Galaxy servers. Connect existing account now to continue to use your existing data and avoid possible loss of data.
```

File: client/src/components/login/Confirmation.vue [hunk 3]
```diff
@@ -0,0 +1,121 @@
...
+                            </p>
+
+                            <b-form-group>
+                                <b-form-checkbox v-model="termsRead">
+                                    I have read and accept these terms.
```
Comment (afgane) [inline client/src/components/login/Confirmation.vue]:
```suggestion
                                    I have read and accept these terms to create a new Galaxy account.
```

**Review by afgane — CHANGES_REQUESTED** [review]
This lgtm, barring the issues with failing tests.

**Review by afgane — APPROVED** [review]

**Comment by dannon** [comment]
It looks like the new `create_user` bits here are duplication and need to be refactored, and I'm not seeing exactly where some of this code is hit anymore, but I know we want to go ahead and get this in and work with it.  We can refine it moving forward.

**Pull request merged**